### DISCUSSION TITLE
Such minimization of comments is annoying

### DISCUSSION THREAD

**Discussion by aleksejrs** [FreeTubeApp/FreeTube#4193] [General]
Somebody minimized [my comment](https://github.com/FreeTubeApp/FreeTube/issues/333#issuecomment-1760979218) about an issue that could impose a limit on the requested feature, without any explanation.

Many "me too" or discussion comments for the feature request itself were randomly minimized or marked as off-topic or duplicate, as if the developers want to get rid of comments.

Issue:  Implement tabs like in web browsers #333
[a comment saying windows would be more useful, ideally having both would be nice](https://github.com/FreeTubeApp/FreeTube/issues/333#issuecomment-595385215) was marked as duplicate. Maybe it is duplicate of [the next comment](https://github.com/FreeTubeApp/FreeTube/issues/333#issuecomment-629852169), which is more detailed and has upvotes.
[The next comment, saying why tabs are better than windows](https://github.com/FreeTubeApp/FreeTube/issues/333#issuecomment-712531088), was marked as off-topic.
[a comment by a now developer discussing windows but calling them tabs](https://github.com/FreeTubeApp/FreeTube/issues/333#issuecomment-845093756) was not hidden.
[the next off-topic comment by the same person](https://github.com/FreeTubeApp/FreeTube/issues/333#issuecomment-925731278) was just minimized.
[three clear me-too comments](https://github.com/FreeTubeApp/FreeTube/issues/333#issuecomment-1478337713) was marked off-topic, and one marked duplicate.

Such confusing feedback discourages commenting.

Hiding of comments with indiscriminate rationales was one of the reasons I stopped contributing to Mozilla.
They switched Firefox to PulseAudio, and a few people complained. When somebody mentioned that apulse could be used as a workaround for the "feature", a developer wrote "PulseAudio maps PulseAudio to ALSA already. We're not intending to remove ALSA support in cubeb or at compile time." (how is a user supposed to understand that?), and all subsequent comments against it were hidden as "advocacy".
It turned out that I had a hardware issue that made PulseAudio crash when playing sound, but now that it and Pipewire kinda work, I am still using ALSA, because those are too complex and apparently not intended for use with multiple accounts.

**Comment by absidue** [comment]
Me too comments add nothing to the conversation and are just spam, having all the spam makes it harder to follow the important information, if you want to add a "me too", please just add a thumbs up to the issue.

Pretty sure multiple Windows already existed, according to Giligus' comment slightly further down, so asking for multiple windows when it already exists and when the issue is about tabs, is off-topic.

The issue is already requesting tabs, the comment saying why tabs are better doesn't add anything more of value.

The comment talking about tabs and windows interchangeably was probably a brain fart, but does make a good point, about how FreeTube will start playback automatically in the new window, even if it isn't focused.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I minimize comments when they don't add anything to the conversation. Comment section should only be used for constructive conversations on how to implement it. Like the ones that are visible.
